Frontend/CO.CDP.OrganisationApp/Pages/Consortium/ConsortiumConfirmSupplier.cshtml

```diff
@@ -0,0 +1,64 @@
+@page "/consortium/{id}/confirm-supplier"
+@model ConsortiumConfirmSupplierModel
+@using CO.CDP.Localization
+
+@{
+    ViewData["Title"] = Model.Heading;
+    var confirmSupplierHasError = ((TagBuilder)Html.ValidationMessageFor(m => m.ConfirmSupplier)).HasInnerHtml;
+    var backLink = $"/consortium/{Model.Id}/enter-sharecode";
+}
+
+@section BeforeContent {
+    <a href="@backLink" class="govuk-back-link">@StaticTextResource.Global_Back</a>
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <partial name="_ErrorSummary" model="@Model.ModelState" />
+        <form class="form" method="post" novalidate>
+            <div class="govuk-form-group @(confirmSupplierHasError ? "govuk-form-group--error": "")">
+                <fieldset class="govuk-fieldset">
+
+                    @if (Request.Query.ContainsKey("frm-summary"))
+                    {
+                    <input type="hidden" name="RedirectToSummary" value="true" />
+                    }
+
+                    <h1 class="govuk-label-wrapper">
+                        <span class="govuk-caption-l">
+                            @Model.ConsortiumName
+                        </span>
+                        <label class="govuk-label govuk-label--l" for="@nameof(Model.ConfirmSupplier)">
+                            @ViewData["Title"]
+                        </label>
+                    </h1>
```

Comment:
This should be a legend not a label since in this instance we are inside a fieldset, and the labels are already associated with the radio buttons down below

```suggestion
                    <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
                        <h1 class="govuk-fieldset__heading">
                            <span class="govuk-caption-l">
                                @Model.ConsortiumName
                            </span>
                            
                            @ViewData["Title"]
                        </h1>
                    </legend>
```